Ports/termcap/patches/0001-Fix-up-some-deprecated-C99-reliances.patch

```diff
@@ -0,0 +1,279 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Tim Schumacher <timschumi@gmx.de>
+Date: Thu, 30 Nov 2023 14:47:40 +0100
+Subject: [PATCH] Fix up some deprecated C99 reliances
```

Comment:
This patch looks generally useful upstream. any activity on it? or does upstream not care about C23 and such

Reply:
Termcap has last been updated in 2002, I strongly doubt that they will take it, much less actually release a new version.